set.mm

```diff
@@ -24945,8 +24945,8 @@ the definition of class equality ( ~ df-cleq ).  Its forward implication
        is called "class extensionality".  Remark: the proof uses ~ axext4 to
        prove also the hypothesis of ~ df-cleq that is a degenerate instance,
        but it could be proved also from minimal propositional calculus and {
-       ~ ax-gen , ~ ax-4 }.  (Contributed by NM, 15-Sep-1993.)  (Revised by BJ,
-       24-Jun-2019.) $)
+       ~ ax-gen , ~ ax-4 , ~ equid }.  (Contributed by NM, 15-Sep-1993.)
```

Comment:
Oh that's simpler than I thought, I guess ax-4 would be removed, then too
(because id ax-gen equid)

Reply:
of course ! I shouldn't do flash edits at midnight !